edb/pgsql/common.py

```diff
@@ -108,7 +108,7 @@ def get_module_backend_name(module, prefix='edgedb_'):
     return edgedb_name_to_pg_name(f'{prefix}{module}', len(prefix))
 
 
-def edgedb_name_to_pg_name(name, prefix_length=0):
+def edgedb_name_to_pg_name(name, prefix_length=0) -> str:
```

Comment:
`name` and `prefix_length` look like easy targets to type, too?

Reply:
Yes.